myokit/formats/sbml/_importer.py

```diff
@@ -96,53 +102,76 @@ def model(self, path):
         refs = {}
 
         # Handle notes, if given
-        x = dom_child(xmodel, 'notes')
+        x = xmodel.find(self.ns['sbml'] + 'notes')
```

Comment:
This won't detect if there's two 'notes' elements, but I suppose we're not super interested in detecting invalid files here...

Reply:
Should we push this issue to the next pull request? I will need to read more closely through the SBML specifications anyway to deal with time, compartments, species etc. and then I can make sure that the file obeys the prescribed format.

Reply:
Yeah it's fine. I'm happy with an importer that just works most of the time, for valid files. There's plenty of validation tools etc. out there already without us adding one :-)